app/graphql/local-schema.gql

```diff
@@ -16,6 +16,7 @@ extend type Query {
   price: String # FIXME test only?
   beta: Boolean!
   colorScheme: String!
+  countryCode: String
```

Comment:
```suggestion
  countryCode: String!
```

if https://github.com/GaloyMoney/galoy-mobile/pull/2895/files#r1438319746 is implemented (no strong opinion but it may help to get cleaner code)

Reply:
updated